File: ide-features-trainer/src/training/ui/LearnCondition.java
package training.ui;

import com.intellij.openapi.project.DumbAware;
import com.intellij.openapi.util.Condition;

/**
 * Created by jetbrains on 17/03/16.
 */
public class LearnCondition implements Condition, DumbAware{

    @Override
    public boolean value(Object o) {
        return false;
    }
}

File: ide-features-trainer/src/training/ui/LearnCondition.kt
/*
 * Copyright 2000-2019 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file.
 */
package training.ui

import com.intellij.openapi.project.DumbAware
import com.intellij.openapi.util.Condition

class LearnCondition : Condition<Any>, DumbAware {

  override fun value(o: Any): Boolean {
    return false
  }
}
